src/clj/rems/application/model.clj

```diff
@@ -572,6 +572,20 @@
       (assoc :application/invited-members (set (vals (:application/invitation-tokens application))))
       (update :application/events (partial mapv #(dissoc % :invitation/token)))))
 
+(defn hide-answers-with-privacy [application roles]
+  (let [fields (get-in application [:application/form :form/fields])
+        fields-with-privacy (mapv (fn [field]
```

Comment:
map with large anonymous function => consider a `for` instead

Reply:
Check the new refactoring.